src/commands/token/spawn.ts

```diff
@@ -12,7 +12,7 @@ export default class Spawn extends Command {
 
   public async run(): Promise<void> {
     const { args } = await this.parse(Spawn)
-    const api = new JamsocketApi()
+    const api = JamsocketApi.fromEnvironment()
```

Comment:
Trying to understand when we are meant to use the `JamsocketApi` class and when we are to use the `Jamsocket` class. Is the idea that for authenticated requests we should use `Jamsocket`, but for unauthed requests we use `JamsocketApi` directly?

Reply:
`JamsocketApi` is intended to be the low-level API wrapping that adds TypeScript annotations to the REST API as it exists. `Jamsocket` is intended to be a JavaScript API that accomplishes actions and does not necessarily map 1:1 to API calls (e.g. push involves both API calls and docker commands).

In cases where the `Jamsocket` would just pass values through to `JamsocketApi`, I've allowed the CLI to interact with `JamsocketApi` directly. If it feels cleaner to just pass everything through `Jamsocket`, I'm open to that, this just seemed to be the pragmatic approach for now.